stanford_mods_config.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
+require 'macros/dlme'
+require 'macros/post_process'
+
+extend Macros::DLME
 extend Macros::IIIF
+extend Macros::PostProcess
```

Comment:
silly question:  is this the only config that wasn't already post processing or using dlme macros?  how odd.

Reply:
@ndushay I'm not sure if it was the *only* config... there may be others that don't use these modules. But they don't need to use these modules until they are changed to do language-tagging.

Adding `Macros::DLME` to a config allows for usage of the `lang()` macro (that operates on a per-field basis), and adding `Macros::PostProcess` allows for usage of the `convert_to_language_hash` macro (that operates on a per-record basis).